configuration.k

```diff
@@ -0,0 +1,220 @@
+
+                // I_H* (block information)
+                // Needed or not? It is part of KEEI, but not sure. Should discuss. 
+                <block>
+                    <hashes>           .List      </hashes>
+                    <coinbase>         0          </coinbase>         // H_c
+                    // <logsBloom>        .WordStack </logsBloom>        // H_b
+                    <difficulty>       0          </difficulty>       // H_d
+                    <number>           0          </number>           // H_i
+                    <gasLimit>         0          </gasLimit>         // H_l
+                    <timestamp>        0          </timestamp>        // H_s
+                </block>
+            </execEngine>
+            
+            // This is the part of the "Solidity VM" where we keep the available contracts. 
+            // Those are either obtained from source file (creation) or loaded from accounts (function call).
+
+            // When we parse Solidity source file we 
+            //   - desugar/preprocess modifiers, inheritance (?) etc. 
+            //   - create one <contract> structure for each contract declared in the sources
+            // (let's not worry about those for now, and just stick to simple functions)
+
+            // From here, <contract>s can be either deployed or executed. 
+            // When a contract is deployed we not only copy its <contract> cell into the <account>, 
+            // but we also copy the other contracts that were present in the same source file, 
+            // so that the deployed contract have access to them and can create and deploy them if needed. 
+            // (see the <account> cell below for more notes about this)
+
+            <contracts>
+                <Contract multiplicity = "*">
+                    <cname> .K </cname>
+                    
+                    <functions>
+                        <Function multiplicity="*">
+                            <fname>         .K         </fname>
+                            <inputParams>  .K         </inputParams> 
+                            <Returns>      .K         </Returns>
+                            <quantifiers>  .K         </quantifiers>
+                            <visibility>   .K         </visibility> // Visibility ::= public | privat
+                            <body> .K </body>
+                        </Function>
+                    </functions>
+		            
+                    <stateVar> .List </stateVar>
+                    <Constructor> false </Constructor>
+                    // ... more to be added later ...
+                
+                </Contract>
+            </contracts>
+
+        </solidity>
+
+        // Ethereum Network
+        // ==========================
+
+        <network>
+
+            // Accounts Record
+            // ---------------
+
+            <activeAccounts> .Set </activeAccounts>
+	        <cntAccount> 0:Int </cntAccount>
```

Comment:
this seems redundant with the `<id>` cell above.

Reply:
Actually, `<cntAccount>` refers to the number of deployed contract instances. I will change the cell name.

Reply:
@shangweilin I'm confused about why this isn't inside the `contract` cell then. If it's tracking how many times a given contract has been deployed, should it be stored with that given contract? Or is it tracking how many contracts have been deployed total?

Reply:
But it's incremented every time that a new `<account>` is made in the rule, even if it has the same contract as another it seems. So I think it will actually just be the same as `size` of `activeAccounts`, not the number of unique contracts deployed.